xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/LikeUIX.xml

```diff
@@ -473,12 +473,14 @@
         #set ($btnClass = "btn-primary")
         #set ($titleMessageTranslationKey = "like.button.title.like")
       #end
-      #set ($disabledButton = "")
+      #set ($disabledButton = false)
       #if (!$services.like.isAuthorized($doc))
-        #set ($disabledButton = "disabled")
+        #set ($disabledButton = true)
         #set ($titleMessageTranslationKey = "like.button.title")
       #end
-      &lt;div class="like-button btn $btnClass $disabledButton badge" title="$escapetool.xml($services.localization.render($titleMessageTranslationKey, [$likeCount]))"&gt;
+      &lt;button class="like-button btn $btnClass badge #if($disabledButton)disabled#{end}"
```

Comment:
Is the ``disabled`` CSS class still needed? Doesn't feel right to have both the CSS class and the attribute.